packages/ui/src/App.tsx

```diff
@@ -30,9 +28,17 @@ const App = () => {
   useEffect(() => {
     (async () => {
       setLoading(true);
-      // TODO: error handling
-      const { settings } = await getSignInExperienceSettings();
+      const { error, settings } = await getSignInExperienceSettings();
+
+      if (error) {
+        setToast('invalid settings'); // TODO: error message
```

Comment:
the page should crash instead?

Reply:
Throw error instead, will check with designer later on a proper error userexperience.